contrib/segmentio/kafka.go.v0/kafka_test.go

```diff
@@ -113,8 +214,8 @@ func TestReadMessageFunctional(t *testing.T) {
 		[]Option{WithDataStreams()},
 	)
 
-	assert.Len(t, writtenMessages, len(messagesToWrite))
-	assert.Len(t, readMessages, len(messagesToWrite))
+	require.Len(t, writtenMessages, len(messagesToWrite))
+	require.Len(t, readMessages, len(messagesToWrite))
```

Comment:
Why switch from `assert` to `require` here? This is the kind of case where `assert` allows further assertions to run which may provide more context about what's happened that caused this to fail.

Reply:
the motivation of the `require` change here is because later in the test these slices are accessed by index, and if the length is not what we expect we will get a panic, making the test output noisy and confusing.